pkg/storage/tableland/impl/tableland/store.go

```diff
@@ -67,7 +67,7 @@ func (s *Store) CreateTable(ctx context.Context, definition tableland.TableDefin
 
 // InsertParts implements InsertParts.
 func (s *Store) InsertParts(ctx context.Context, parts []local.Part) error {
-	tableName, err := s.localStore.TableName(ctx, "parts", s.chainID)
```

Comment:
Transform data stored in our local SQLite db into SQL insert statements to populate our new rigs table on Tableland.